config/v1/types_infrastructure.go

```diff
@@ -313,6 +313,18 @@ type AWSPlatformStatus struct {
 	// There must be only one ServiceEndpoint for a service.
 	// +optional
 	ServiceEndpoints []AWSServiceEndpoint `json:"serviceEndpoints,omitempty"`
+
+	// userTags is a list of additional tags to apply to AWS resources created for the cluster.
+	// +optional
+	UserTags []AWSUserTag `json:"userTags,omitempty"`
```

Comment:
how many do we want to allow. AWS only allows 50, how many should we preserve for ourselves?